.nuspec/Xamarin.Forms.nuspec

```diff
@@ -207,9 +206,13 @@
     <file src="..\Stubs\Xamarin.Forms.Platform.Android\bin\$Configuration$\MonoAndroid10.0\Xamarin.Forms.Platform.dll" target="lib\MonoAndroid10.0" />
 
     <!--iPhone Unified-->
-    <file src="..\Xamarin.Forms.Platform.iOS\bin\$Configuration$\Xamarin.Forms.Platform.iOS.dll" target="lib\Xamarin.iOS10" />
```

Comment:
If you remove this line, then does it install into ios? Or is it picking the netstandard dll?

Reply:
This dll isn't referenced anywhere else in the nuspec so it there isn't a NS version that comes in. Since the dll doesn't even exist in the lib folder anymore I had to remove this for the nuget to even package